cmd/guacone/cmd/files.go

```diff
@@ -178,12 +185,13 @@ var exampleCmd = &cobra.Command{
 	},
 }
 
-func validateFlags(user string, pass string, dbAddr string, realm string, keyPath string, keyID string, args []string) (options, error) {
+func validateFlags(user string, pass string, dbAddr string, realm string, keyPath string, keyID string, graphqlEndpoint string, args []string) (options, error) {
 	var opts options
 	opts.user = user
 	opts.pass = pass
 	opts.dbAddr = dbAddr
 	opts.realm = realm
+	opts.graphqlEndpoint = graphqlEndpoint
```

Comment:
Should we build this based on `opts.port` or validate that the `gql-endpoint` config includes the `gql-port` one?

Reply:
it's kind of weird cause they are actually totally separate entities that are using them... we can validate it but anything that isn't running the server wouldn't actually need to configure it.... I think this is a design decision we have to make. At the moment it's a bit messy, and we have overlapping cmds as well.

Reply:
i'll open an issue for now to discuss this https://github.com/guacsec/guac/issues/531

Reply:
Seems good to punt for #531